CHANGELOG.md

```diff
@@ -11,6 +11,14 @@ but this project DOES NOT adhere to [Semantic Versioning](http://semver.org/).
 * Remove `LeanCode.AsyncTasks` and Hangfire-based implementation
 * `FindAndEnsureExistsAsync` now throws `EntityDoesNotExistException`
 * Remove `AddAsync`, `UpdateAsync`, `DeleteAsync` and `DeleteRangeAsync` from `EFRepository` and `IRepository`
+* Remove `IRepository<T>`
+* Change `EFRepository.Update` so that it does not call `dbContext.Update` anymore
+* Rework `FixedTimeProvider`
+* Add query's and operation's responses in logs
```

Comment:
It adds a middleware (aka element) for doing that. It is worth mentioning, otherwise you might think that it is always enabled.